modules/checking-cluster-resources-after-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-cluster-resources-after-initiating-limited-live-migration_{context}"]
+= Checking cluster resources after initiating the limited live migration
+
+The following procedure shows you how to check metrics for EgressIPs resources, EgressFirewalls resources, and multicast enabled namespaces. If you have these resources, you should check them after migration to ensure their intended behavior.
```

Comment:
The following procedure does not check the metrics. The metrics should be checked from the OpenShift Container Platform web console, navigate to Observe -> Metrics.

Reply:
Changed to "The following procedure shows you how to check for EgressIPs resources. . ."